build.sbt

```diff
@@ -27,6 +28,10 @@ lazy val expecty = (projectMatrix in file("."))
   .settings(
     name := "Expecty",
     scalacOptions ++= Seq("-Yrangepos", "-feature", "-deprecation"),
+    scalacOptions ++= (CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((3, _)) => Seq("-scala-output-version", "3.1")
+      case _            => Nil
+    }),
```

Comment:
I thought the output version is no longer a thing - https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html

Reply:
@eed3si9n ergh, the dotty team should consider updating the [docs](https://docs.scala-lang.org/scala3/reference/language-versions/binary-compatibility.html) to reflect that. 

Anyhow, are we cool with bumping to 3.2 then ?

Reply:
Do we need to bump to 3.2.0?

3.1.0 should be fine for both native and CE.

I'm always team upgrade, but I think in this case there is no reason - CE is going to 3.2.0 because there's a physically unfixable issue with 3.1 bytecode that CE triggers

Reply:
that's fair enough, as long as I'm not the one making the decision I'm up for everything and anything 😄